src/teletype.h

```diff
@@ -8,8 +8,8 @@
 #include "command.h"
 #include "state.h"
 
-#define TELETYPE_VERSION "TELETYPE 2.1.0"
 #define TELE_ERROR_MSG_LENGTH 16
+#define TELETYPE_PROFILE // un-comment this line to enable profiling
```

Comment:
profiling is *on* then, right?

should probably turn this off for release.

Reply:
Oops!  Yes, comment that out for release.